homeassistant/components/nest/__init__.py

```diff
@@ -214,7 +219,7 @@ async def async_config_reload() -> None:
     except SubscriberException as err:
         if DATA_NEST_UNAVAILABLE not in hass.data[DOMAIN]:
             _LOGGER.error("Subscriber error: %s", err)
-            hass.data[DOMAIN][DATA_NEST_UNAVAILABLE] = True
+            hass.data[DOMAIN][entry.entry_id][DATA_NEST_UNAVAILABLE] = True
```

Comment:
Storing things in hass.data before raising ConfigEntryNotReady is unconventional.

Reply:
My impression is this used to be the way to meet higher integration quality scales, then someone refactored to improve via new exception handling, and this was just left here. Removed.